packages/terra-search-field/src/SearchField.module.scss

```diff
@@ -95,6 +95,19 @@
     }
   }
 
+  .label {
+    color: var(--terra-search-field-label-color, #1c1f21);
```

Comment:
I think we could also add this var in dark theme (I think it would be #b2b5b6, Orion Fusion has the same value so not sure if we need to specify anyways there)
https://github.com/cerner/terra-core/blob/71c926b2d981ab5ffbe8c62e9b8ea3bb5529be88/packages/terra-alert/src/clinical-lowlight-theme/Alert.module.scss#L16C46-L16C46